src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx

```diff
@@ -137,11 +137,11 @@ SELECT function(attribute)
       <td id="facet">
         `FACET attribute`
 
-        **Required for outlier conditions, but not baseline or static**
+        **Required for outlier conditions.**
```

Comment:
Maybe we don't need the period here since the other bold entries don't have one.